.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "tempesta-tech.com"]
+	path = tempesta-tech.com
+	url = https://github.com/tempesta-tech/tempesta-tech.com.git
```

Comment:
We should use git@github.com:tempesta-tech/tempesta-tech.com.git here and in .git/config

Reply:
> I left authentication by https because our CI works with https  auth.